client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -8,6 +8,9 @@ import Typography from '@material-ui/core/Typography';
 import useStyles from './useStyles';
 import { CircularProgress } from '@material-ui/core';
 
+import {useAuth} from '../../../context/useAuthContext';
+import login from '../../../helpers/APICalls/login';
+
```

Comment:
Line 135, the current <Link> component directs to './signup'  please change to redirect to the 'login form'

Reply:
corrected